### PR TITLE
Add extra resources support

### PR DESCRIPTION
**Pull request by mikheillomidzeq**
Add extra resources support

### CODE REVIEW

File: qdrant/resource_accounts_cluster_test.go [hunk 1]
```diff
@@ -121,3 +121,79 @@ output "cluster_cfg_num_nodes" {
...
+	`, os.Getenv("QDRANT_CLOUD_API_KEY"))
+
+	config := provider + fmt.Sprintf(`
+data "qdrant-cloud_booking_packages" "test" {
+  cloud_provider = "gcp"
+  cloud_region   = "us-east4"
+}
+locals {
+  resource_data = data.qdrant-cloud_booking_packages.test.packages
+  // Filter out the gpx1
```
Comment (Robert-Stam) [inline qdrant/resource_accounts_cluster_test.go:139]: filter out is double imo: Filter out the gpx1 --> Filter on gpx1

File: qdrant/schemas_accounts_clusters.go [hunk 2]
```diff
@@ -169,6 +173,14 @@ func accountsClusterNodeConfigurationSchema(asDataSource bool) map[string]*schem
 			Required:    !asDataSource,
 			Computed:    asDataSource,
 		},
+		resourceConfigurationsFieldName: {
+			Description: descriptionResourceConfigurations,
+			Type:        schema.TypeList,
+			Optional:    true,
+			Elem: &schema.Resource{
+				Schema: resourceConfigurationsSchema(false),
```
Comment (Robert-Stam) [inline qdrant/schemas_accounts_clusters.go:181]: false --> asDataSource

**Review by Robert-Stam — COMMENTED** [review]
2 very small things, further LGTM.

PS: Personally I would have opted for a more explicit additionalDisksInGi to the (public) API and in the TF state, now it's very flexible, but the customers need to know what to do exactly.

File: qdrant/resource_accounts_cluster_test.go (hunk 1 above)
Comment (mikheillomidzeq) [reply qdrant/resource_accounts_cluster_test.go]: Fixed [here](https://github.com/qdrant/terraform-provider-qdrant-cloud/commit/0fda97fd6f3b8b48377a79a85ce23ca2d4909e74)

File: qdrant/schemas_accounts_clusters.go (hunk 2 above)
Comment (mikheillomidzeq) [reply qdrant/schemas_accounts_clusters.go]: fixed [here](https://github.com/qdrant/terraform-provider-qdrant-cloud/commit/fe069a45a95774d9f95c2a33519f1250a236640e)

**Review by Robert-Stam — APPROVED** [review]
LGTM

**Pull request merged**